From 291d089142db69e85a78a95ef1680552f4393b81 Mon Sep 17 00:00:00 2001
From: Duncan Macleod <duncan.macleod@ligo.org>
Date: Mon, 29 Apr 2024 10:52:45 +0100
Subject: [PATCH] remove lalsuite from requirements

in conda we can depend on just the subpackages we need
---
 pyproject.toml | 1 -
 setup.cfg      | 1 -
 2 files changed, 2 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 17526ae..74562b5 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,7 +1,6 @@
 [build-system]
 requires = [
     "cython",
-    "lalsuite",
     "numpy",
     "setuptools",
     "wheel",
diff --git a/setup.cfg b/setup.cfg
index b0c4de3..faa7e9a 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -36,7 +36,6 @@ setup_requires =
 	setuptools >= 30.3.0
 install_requires =
 	h5py
-	lalsuite
 	lscsoft-glue
 	matplotlib
 	numpy
-- 
2.39.2

